test/safe/pull-request.test.ts

```diff
@@ -1,63 +1,312 @@
-import { mocked } from "ts-jest/utils";
 /* eslint-disable @typescript-eslint/no-var-requires */
 import { createWebhookApp } from "../utils/probot";
 import { Application } from "probot";
 import { Installation, Subscription } from "../../src/models";
-import nock from "nock";
 
-jest.mock("../../src/models");
-
-describe("GitHub Actions", () => {
+describe("Pull Request Webhook", () => {
 	let app: Application;
+	const gitHubInstallationId = 1234;
+
 	beforeEach(async () => {
 		app = await createWebhookApp();
+		const clientKey = "client-key";
+		await Installation.create({
+			clientKey,
+			sharedSecret: "shared-secret",
+			jiraHost
+		});
+		await Subscription.create({
+			gitHubInstallationId,
+			jiraHost,
+			jiraClientKey: clientKey
+		});
 	});
 
 	afterEach(async () => {
-		if (!nock.isDone()) {
```

Comment:
I wonder did we move it to some kind of global verification?

Reply:
Yep! It's in `test/setup.ts`